ci-operator/jobs/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master-presubmits.yaml

```diff
@@ -297,6 +297,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )openstack-operator-build-deploy,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Arx, I thought we decied moving to "always_run: false"

[1] https://github.com/openshift/release/pull/35881

Reply:
You right, my mistake, changing